src/display/svg.js

```diff
@@ -1200,8 +1158,30 @@ var SVGGraphics = (function SVGGraphicsClosure() {
     },
 
     paintFormXObjectEnd:
-        function SVGGraphics_paintFormXObjectEnd() {
-      this.restore();
+        function SVGGraphics_paintFormXObjectEnd() {},
+
+    /**
+     * @private
+     */
+    _ensureTransformGroup: function SVGGraphics_ensureTransformGroup() {
+      if (!this.tgrp) {
+        this.tgrp = document.createElementNS(NS, 'svg:g');
+        this.tgrp.setAttributeNS(null, 'transform', pm(this.transformMatrix));
+      }
+    },
+
+    /**
+     * @private
+     */
+    _finishGroup: function SVGGraphics_finishGroup() {
```

Comment:
_finishGroup can be removed, see https://github.com/yurydelendik/pdf.js/commit/f4c8099678a0d19f1e48c891aa55c2581931495f